frontend/src/app/main/data/workspace.cljs

```diff
@@ -1197,10 +1224,16 @@
       (watch [_ state stream]
         (let [objects  (dwc/lookup-page-objects state)
               selected (get-in state [:workspace-local :selected])
-              cdata    (prepare-selected objects selected)]
-          (->> (t/encode cdata)
-               (wapi/write-to-clipboard)
-               (rx/from)
+              pdata    (reduce (partial collect-object-ids objects) {} selected)
+              initial  {:type :copied-shapes
+                        :selected selected
+                        :objects {}
+                        :images #{}}]
+          (->> (rx/from (seq (vals pdata)))
+               (rx/fmap (partial prepare-object objects selected))
```

Comment:
I think using the rxjs name  `merge-map` it's better for legibility. I did have to go to the source code to see what `fmap` meant.